data/fields/fire_mains.json

```diff
@@ -0,0 +1,15 @@
+{
+    "key": "fire_mains",
+    "type": "combo",
+    "label": "Type",
+    "strings": {
+        "options": {
+            "dry": "Dry Riser Inlet",
+            "wet": "Charged Riser",
+            "yes": "yes",
```

Comment:
as the `yes` value is [not](https://taginfo.openstreetmap.org/keys/fire_mains#values) [used](https://wiki.openstreetmap.org/wiki/Key%3Afire_mains)) for this tag, I think we should just omit it.

Reply:
//edit: oh… now I see that the [proposal](https://wiki.openstreetmap.org/w/index.php?oldid=2483791) did also introduce the `yes` option. In that case it would be best to label it _"Unknown Type"_.